.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -12,13 +17,29 @@ What did you expect to happen? What actually happened?
 -->
 
 ## Minimal Code Snippet
-<!-- Please provide example code that demonstrates the issue. Do NOT paste screenshots of code! -->
+<!-- Please provide example code that allows to reproduce the issue. Do NOT paste screenshots of code! -->
```

Comment:
```suggestion
<!-- Please provide example code that allows anyone to reproduce the issue. Do NOT paste screenshots of code! -->
```

Reply:
Well, _anyone_ may be expecting a bit much... ?

Reply:
Feel free to change to "us". :-)